src/cdk/activation.ts

```diff
@@ -22,9 +25,33 @@ export async function activate(activateArguments: { extensionContext: vscode.Ext
         treeDataProvider: explorer,
         showCollapseAll: true
     })
+
+    // Indicates workspace includes a CDK app and user has expanded the Node
+    view.onDidExpandElement(e => {
+        if (e.element.contextValue === 'awsCdkAppNode') {
+            ext.telemetry.record(getTelemetryEvent('appExpanded'))
+        }
+    })
+
+    // Indicates CDK explorer was disabled
+    vscode.workspace.onDidChangeConfiguration(e => {
+        const explorerEnabled = 'aws.cdk.explorer.enabled'
+        if (e.affectsConfiguration(explorerEnabled) && !vscode.workspace.getConfiguration().get(explorerEnabled)) {
+            ext.telemetry.record(getTelemetryEvent('explorerDisabled'))
```

Comment:
Would you mind making all of the event names constants at the top of the file? The current code doesn't do this unfortunately, so there hasn't been a good example to base this on.

Reply:
shifted it to an enum, extracted out constants.
I could also move the telemetry initialization into it's own method maybe.. to keep the activate method light...